pkg/acsengine/testdata/disks-managed/swarm-vmas_expected.json

```diff
@@ -507,6 +507,13 @@
         "description": "SSH public key used for auth to all Linux machines.  Not Required.  If not set, you must provide a password key."
       },
       "type": "string"
+    },
+    "targetEnvironment": {
+      "defaultValue": "AzurePublicCloud",
+      "metadata": {
+        "description": "The azure deploy environment. eg. AzurePublicCloud, AzureChinaCloud, AzureGermanCloud, AzureUSGovermentCloud"
```

Comment:
for the sake of consistency with azure powershell, these environments are
AzureCloud, AzureChinaCloud, AzureUSGovernment, and AzureGermanCloud.